pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -211,53 +207,26 @@ type OFBridge struct {
 	tunMetadataLengthMap map[uint16]uint8
 }
 
-func (b *OFBridge) CreateGroupTypeAll(id GroupIDType) Group {
+func (b *OFBridge) NewGroupTypeAll(id GroupIDType) Group {
 	return b.createGroupWithType(id, ofctrl.GroupAll)
 }
 
-func (b *OFBridge) CreateGroup(id GroupIDType) Group {
+func (b *OFBridge) NewGroup(id GroupIDType) Group {
 	return b.createGroupWithType(id, ofctrl.GroupSelect)
 }
 
 func (b *OFBridge) createGroupWithType(id GroupIDType, groupType ofctrl.GroupType) Group {
```

Comment:
I think this function name should be changed too to be consistent.

Reply:
updated.